api/clients/v2/relay_client.go

```diff
@@ -52,6 +54,20 @@ type RelayClient interface {
 	Close() error
 }
 
+// grpcClient encapsulates a gRPC client and a channel for limiting concurrent requests to that client.
+type grpcClient struct {
```

Comment:
What do you think of the name `limitedClient`, or maybe `limitedConcurrencyClient`? 

Not sure if these are good suggestions, but `grpcClient` doesn't indicate the reason this struct exists

Reply:
renamed to `limitedClient`